Before datagen..
Found 2800 images belonging to 2 classes.
Datagen completed..
Found 800 images belonging to 2 classes.
Data Generation Completed
Model: "sequential"
_________________________________________________________________
 Layer (type)                Output Shape              Param #
=================================================================
 conv2d (Conv2D)             (None, 252, 252, 32)      2432
 max_pooling2d (MaxPooling2D  (None, 126, 126, 32)     0
 )
 batch_normalization (BatchN  (None, 126, 126, 32)     128
 ormalization)
 conv2d_1 (Conv2D)           (None, 124, 124, 64)      18496
 max_pooling2d_1 (MaxPooling  (None, 62, 62, 64)       0
 2D)
 batch_normalization_1 (Batc  (None, 62, 62, 64)       256
 hNormalization)
 conv2d_2 (Conv2D)           (None, 60, 60, 64)        36928
 max_pooling2d_2 (MaxPooling  (None, 30, 30, 64)       0
 2D)
 conv2d_3 (Conv2D)           (None, 28, 28, 128)       73856
 max_pooling2d_3 (MaxPooling  (None, 14, 14, 128)      0
 2D)
 flatten (Flatten)           (None, 25088)             0
 dense (Dense)               (None, 128)               3211392
 dense_1 (Dense)             (None, 1)                 129
=================================================================
Total params: 3,343,617
Trainable params: 3,343,425
Non-trainable params: 192
_________________________________________________________________
/home/akshat/Documents/Github/homo_env/lib/python3.10/site-packages/keras/optimizers/optimizer_v2/rmsprop.py:135: UserWarning: The `lr` argument is deprecated, use `learning_rate` instead.
  super(RMSprop, self).__init__(name, **kwargs)
[34m[1mwandb[39m[22m: [33mWARNING[39m The save_model argument by default saves the model in the HDF5 format that cannot save custom objects like subclassed models and custom layers. This behavior will be deprecated in a future release in favor of the SavedModel format. Meanwhile, the HDF5 model is saved as W&B files and the SavedModel as W&B Artifacts.
Traceback (most recent call last):
  File "/home/akshat/Documents/Github/Homoglyph_Phishing_Detection/code/train.py", line 221, in <module>
    obj.SimpleCNN(conbfig)
  File "/home/akshat/Documents/Github/Homoglyph_Phishing_Detection/code/train.py", line 81, in SimpleCNN
    WandbCallback(monitor="vall_loss", save_mode=(True), log_weights=(True), log_gradients=(True))
  File "/home/akshat/Documents/Github/homo_env/lib/python3.10/site-packages/wandb/integration/keras/keras.py", line 468, in __init__
    raise ValueError(
ValueError: training_data argument is required for gradient logging.